oemoflex/postprocessing.py

```diff
@@ -91,6 +115,10 @@ def fnc(flow):
         # preserve its content ("invest" for now)
         endogenous.rename(columns={"type": "var_name"}, inplace=True)
 
+        # Update "var_name" with Storage specific parameter names for charge and discharge devices
```

Comment:
Couldn't this be done the same as in the next lines? Like
endogenous["var_name"] = endogenous.apply(get_parameter_name, axis=1)